ngrep.c

```diff
@@ -395,6 +400,10 @@ int main(int argc, char **argv) {
             memset(&mask, 0, sizeof(mask));
         }
 
+        if ((!strncasecmp(dev, "lo", 2)) || (!strcasecmp(dev, "any"))) {
```

Comment:
May be accidental, but I think this is actually the solution to [this problem](https://bugzilla.redhat.com/show_bug.cgi?id=1401152) too - VLAN + Linux-SLL doesn't work.  Thanks!